apps/core/tailwind.config.js

```diff
@@ -92,6 +92,7 @@ module.exports = {
         subtitle: ["12px", "1"],
         subtitleSmall: ["11px", "1"],
         subtitleSmallExtra: ["10px", "1"],
+        extraSmall: '8px',
```

Comment:
I don't see this in the design system under typography?

Reply:
I am removing this. It was a temp for the fonts icon. Once the icon PR lands, I will remove all fontsIcon